pkg/authorization/hardcodedauthorizer/tokenreview.go

```diff
@@ -0,0 +1,41 @@
+package hardcodedauthorizer
+
+import (
+	"context"
+
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+type tokenReviewAuthorizer struct{}
```

Comment:
why is this in library-go?

Reply:
> why is this in library-go?

only because it was easy to write the tests here.  I'm not particularly attached.